tox.ini

```diff
@@ -20,6 +20,7 @@ skip_install = true
 deps =
     setuptools < 52
     zc.buildout
+    zope.testrunner
```

Comment:
Is this necessary?

Reply:
https://github.com/zopefoundation/Products.ZMySQLDA/pull/26/commits/88c89f6863ec09389a9834bb34c654005f605b42 tries without that dependency.

Reply:
This strangely seems to fix the `py27-zope2` env.